job/server/src/main/java/alluxio/job/plan/transform/format/SchemaConversionUtils.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.job.plan.transform.format;
+
+import alluxio.job.plan.transform.FieldSchema;
+import alluxio.job.plan.transform.HiveConstants;
+import alluxio.job.plan.transform.format.csv.CsvUtils;
+import alluxio.job.plan.transform.format.csv.Decimal;
+
+import org.apache.avro.LogicalTypes;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility Class for converting schema to Parquet.
+ */
+public class SchemaConversionUtils {
```

Comment:
were these methods mainly copied from the previous location?

Reply:
Yep. These were mainly copied